doc/classes/ParallaxLayer.xml

```diff
@@ -60,10 +61,13 @@
 	</methods>
 	<members>
 		<member name="motion_mirroring" type="Vector2" setter="set_mirroring" getter="get_mirroring">
+			 Mirroring of the [code]ParallaxLayer[/code], reapeat the [code]ParallaxLayer[/code]. Useful to create an infinite scrolling background. If an axis is set to 0 then that axis will have no mirroring. 
 		</member>
 		<member name="motion_offset" type="Vector2" setter="set_motion_offset" getter="get_motion_offset">
+			 Start offset of the [code]ParallaxLayer[/code].
 		</member>
 		<member name="motion_scale" type="Vector2" setter="set_motion_scale" getter="get_motion_scale">
+			Motion scale of the [code]ParallaxLayer[/code]. If an axis is set to 0 then it will not move and it will stick with the camera.
```

Comment:
Multiplies the ParallaxLayer's motion. If an axis is set to [code]0[/code] it will not scroll.